extensions/amp-story/1.0/amp-story.js

```diff
@@ -355,8 +355,8 @@ export class AmpStory extends AMP.BaseElement {
   pauseCallback() {
     // Store the current paused state, to make sure the story does not play on
     // resume if it was previously paused.
-    this.pausedStateToRestore_ =
-        this.storeService_.get(StateProperty.PAUSED_STATE);
+    this.pausedStateToRestore_ = /** @type {boolean} */ (
+      this.storeService_.get(StateProperty.PAUSED_STATE));
```

Comment:
Nit: Would `!!this.storeService_.get(StateProperty.PAUSED_STATE)` be safer? It'd satisfy the compiler, and also ensure it's a boolean.

Reply:
yeah, i think that would be safer